endpoints/openrtb2/sample-requests/amp/consent-through-query/addtl-consent-through-query.json

```diff
@@ -105,8 +105,7 @@
             ]
           },
           "includewinners": true,
-          "includebidderkeys": true,
-          "mediatypepricegranularity": {}
```

Comment:
Changed `mediatypepricegranularity` and `publisher` to be omitted when empty. Not sure why this wasn't done initially. I doubt it was intentional.